alf/algorithms/ddpg_algorithm.py

```diff
@@ -157,10 +159,13 @@ def _sample(a, ou):
                 tf.less(tf.random.uniform((), 0, 1),
                         epsilon_greedy), lambda: a + ou(), lambda: a)
 
-        action = tf.nest.map_structure(_sample, action, self._ou_process)
+        noisy_action = tf.nest.map_structure(_sample, action, self._ou_process)
```

Comment:
should clip noise action  to match action spec

Reply:
fixed.